.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
woow! i did not that was possible. honestly tho i do not have a good understanding of how babel works and the difference between presets and plugins all i know about babel is that i need babel-core and presets2015 to compile code to es6 :) anyway i had to go through your links and babel docs to get a better understanding. 

BTW i wonder do we need ES6 modules plugin? since webpack can take care of that for us and as you said that we should not use es6 modules on the server so what do we need it for?